url.bs

```diff
@@ -823,6 +842,27 @@ unified model would be, please file an issue.
 <!-- History behind URL as term:
      https://lists.w3.org/Archives/Public/uri/2012Oct/0080.html -->
 
+<p>At a high level, a <a for=/>URL</a>, <a>URL string</a>, <a>URL parser</a>, and
```

Comment:
Related: https://github.com/whatwg/html/pull/2318

Since HTML uses "valid URL" (and "valid foo" in general), maybe URL spec should use "valid URL string", and HTML can use that term directly?